R/dimensions.R

```diff
@@ -101,3 +102,19 @@ pitch_international <- list(
   origin_x = 0,
   origin_y = 0
 )
+
+#' @rdname pitch_tracab
```

Comment:
You've added mention of tracab to the main docs and `pitch_tracab` doesn't have any documentation attached, which makes me think this should be `@rdname pitch_opta`, instead. Is that right?